src/cdk/testing/testbed/unit-test-element.ts

```diff
@@ -241,10 +248,9 @@ export class UnitTestElement implements TestElement {
     }
 
     this._dispatchPointerEventIfSupported('pointerdown', clientX, clientY, button);
-    dispatchMouseEvent(this.element, 'mousedown', clientX, clientY, button);
+    dispatchMouseEvent(this.element, 'mousedown', clientX, clientY, button, modifiers);
     this._dispatchPointerEventIfSupported('pointerup', clientX, clientY, button);
-    dispatchMouseEvent(this.element, 'mouseup', clientX, clientY, button);
-    dispatchMouseEvent(this.element, name, clientX, clientY, button);
-    await this._stabilize();
+    dispatchMouseEvent(this.element, 'mouseup', clientX, clientY, button, modifiers);
+    dispatchMouseEvent(this.element, name, clientX, clientY, button, modifiers);
```

Comment:
why was `await this._stabilize()` removed?

Reply:
`_stabilize` is already called from the public entry points `click` and `rightClick` that use this private helper. It's not necessary to call it more often.

Reply:
can we add a second `await this._stabilize()` to `click` to maintain the same behavior that it has currently? I tried presubmitting and it seems maybe some people were depending on that quirk. You can leave a TODO to clean it up later, just don't want it to block this

Reply:
Obviously I can, but that second `await this._stabilize()` was recently added in https://github.com/angular/components/pull/20400 which makes that there cannot be many people, if any at all, relying on it

Reply:
It's possible the failures I saw were not related to this line. I'll do some more digging and let you know

Reply:
I confirmed that removing that call to `this._stabilize()` was indeed the culprit. It'll need to be investigated, but shouldn't block this PR. Can you add it back with a comment to investigate why removing it breaks some tests in g3?

Reply:
Will do